README.md

```diff
@@ -43,7 +43,7 @@ echo shhh-dont-tell-anyone > secrets/local-config-secret.txt
 echo "secrets/*.txt" >> .gitignore
 ```
 
-Second, we let confair know about the secret with some metadata:
+Second, let confair know about the secret with some metadata:
```

Comment:
"First, create a file"
"Second, let confair"

OR

"First we create a file"
"Second, we let confair"